src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -120,52 +129,65 @@ const styles = StyleSheet.create({
-            </View>
-            <View style={[styles.tableColumn, styles.instructionsColumnWidth]}>
-                <Text>Instructions</Text>
-            </View>
-            <View style={[styles.tableColumn, styles.numberOfLabelsColumnWidth]}>
-                <Text>Number of Parcels</Text>
+    const createHeader = (category: CreateHeaderProps): React.JSX.Element => {
+        return (
+            <View style={[styles.tableHeader, styles.flexRow]}>
+                <View style={[styles.tableColumn, styles.nameColumnWidth]}>
+                    <Text>Name</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.addressColumnWidth]}>
+                    <Text>{category === "Collection" ? "Collection Centre" : "Address"}</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.contactColumnWidth]}>
+                    <Text>Contact</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.packingDateColumnWidth]}>
+                    <Text>Packing Date</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.instructionsColumnWidth]}>
+                    <Text>Instructions</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.numberOfLabelsColumnWidth]}>
+                    <Text>Number of Parcels</Text>
+                </View>
             </View>
-        </View>
-    );
+        );
+    };
 
-    const table = data.tableData.map((rowData, index) => {
+    const createRowData = (rowData: DriverOverviewTableData, index: number): React.JSX.Element => {
         return (
             // eslint-disable-next-line react/no-array-index-key
             <View key={index} style={[styles.tableRow, styles.flexRow]} wrap={false}>
                 <View style={[styles.tableColumn, styles.nameColumnWidth]}>
                     <Text>{rowData.name}</Text>
                 </View>
                 <View style={[styles.tableColumn, styles.addressColumnWidth]}>
-                    {rowData.address.postcode || rowData.clientIsActive ? (
+                    {rowData.collection_centre.isDelivery ? (
+                        rowData.address.postcode || rowData.clientIsActive ? (
+                            <>
+                                <Text>{rowData.address.line1}</Text>
+                                <Text>{rowData.address.line2}</Text>
+                                <Text>{rowData.address.town}</Text>
+                                <Text>{rowData.address.county}</Text>
+                                <Text>{rowData.address.postcode}</Text>
+                            </>
+                        ) : (
+                            <Text>
+                                {rowData.clientIsActive ? displayPostcodeForHomelessClient : "-"}
+                            </Text>
+                        )
+                    ) : (
                         <>
```

Comment:
Do the collection centres have addresses? From the ticket "Group Collections and mark them with the name of the collection centre and the Feet Icon." I interpreted that as this structure

Deliveries van_icon
table

Collection - centre 1 feet_icon
table

Collection - centre 2 feet_icon
table
...

in which case it might be sensible to keep the address and put in the one of the collection centre itself

Although I do prefer your implementation to that and if the collection centres don't have an address it's probably best to keep as is